src/rcx/include/rcx/extRCap.h

```diff
@@ -717,8 +717,8 @@ class extMeasure
   bool verticalCap(int rsegId1,
                    uint rsegId2,
                    uint len,
-                   uint diagDist,
                    uint tgtWidth,
+                   uint diagDist,
```

Comment:
This seems like a risky change.

Reply:
The header and the cpp were opposite!  I changed the header to match the cpp and verified the call site.